src/components/Header/HeaderUser.jsx

```diff
@@ -0,0 +1,201 @@
+                            <PointsBalance>
+                                Points balance:
+                                <NumberBalance>{userBalance}</NumberBalance>
+                            </PointsBalance>
+                        }
+                    </LeftBlock>
+                    <MenuHeaderBlock>
+                        <NavLink to="/" style={linkActiveColorDesktop} >
+                            <MenuSection>Home</MenuSection>
+                        </NavLink>
+                        <NavLink to="planning" style={linkActiveColorDesktop} >
+                            <MenuSection>Planning</MenuSection>
+                        </NavLink>
+                        <NavLink to="awards" style={linkActiveColorDesktop} >
+                            <MenuSection>Awards</MenuSection>
+                        </NavLink>
+                        <NavLink to="contacts" style={linkActiveColorDesktop} >
+                            <MenuContact>Contacts</MenuContact>
+                        </NavLink>
+                        <MenuInitiaLetterCircle>
+                            <MenuInitiaLetter>{nameLetter}</MenuInitiaLetter>
+                        </MenuInitiaLetterCircle>
+                        <MenuName>{userName}</MenuName>
+                        <NavLink to="auth" onClick={logOut}>
+                            <ExitIconWrapper src={ExitIcon} alt="ExitIcon" />
+                        </NavLink>
+                    </MenuHeaderBlock>
+                </ContainerHeader>
+            }
+            {(width >= 530 && width <= 869) &&   // --------------- Tablet -------------
+                <>
+                    <ContainerHeader>
+                        <LeftBlock>
+                            <LogoHead />
+                            <PointsBalance>
+                                Points balance:
+                                <NumberBalance>{userBalance}</NumberBalance>
+                            </PointsBalance>
+                        </LeftBlock>
+                        <MenuHeaderBlock>
+                            <MobileHamburgerContainer onClick={toggle} >
+                                <MobileHamburgerLine />
+                            </MobileHamburgerContainer>
+                            <MenuInitiaLetterCircle>
+                                <MenuInitiaLetter>{nameLetter}</MenuInitiaLetter>
+                            </MenuInitiaLetterCircle>
+                            <MenuName>{userName}</MenuName>
+                            <NavLink to="auth" onClick={logOut}>
+                                <ExitIconWrapper src={ExitIcon} alt="ExitIcon" />
+                            </NavLink>
+                        </MenuHeaderBlock>
+                    </ContainerHeader>
+                    {isOpen &&
+                        <MobileHamburgerDarkened >
+                        </MobileHamburgerDarkened>
+                    }
+                    <MobileHamburgerbackground Active={isOpen}>
```

Comment:
To make this component smaller you could make some standalone components inside this folder.
Like: Hamburger, Navigation, PointsBalance and etc

Reply:
Most likely I can separate the following blocks into separate components:
- MobileHamburgerLoginBox
- MobileHamburgerCrossBox
- Navigation (NavLink)